spec/models/card_spec.rb

```diff
@@ -134,14 +134,14 @@
     card = Card.create(original_text: 'дом', translated_text: 'house',
                        block_id: 1)
     expect(card.errors[:user_id]).
-        to include('Ошибка ассоциации.')
+        to include('Заполните поле.')
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.